src/sidebar/templates/group-list-section.html

```diff
@@ -0,0 +1,13 @@
+<h2 class="heading">
+  {{ vm.heading }}
+</h2>
+<ul class="content">
+  <li class="dropdown-menu__row dropdown-menu__row--unpadded"
+      ng-repeat="group in vm.sectionGroups track by group.id">
+
+    <group-list-item class="group-list-item" group="group" ng-if="vm.isSelectable(group.id)"></group-list-item>
+
+    <group-list-out-of-scope-item class="group-list-out-of-scope-item" group="group" ng-if="!vm.isSelectable(group.id)"></group-list-out-of-scope-item>
```

Comment:
Small request, can you format this onto multiple lines so that it is easier to review. For new template files, you can use the same [Prettier](https://prettier.io) formatter that we use for JavaScript for HTML files. If you have the Vim plugin installed, this can be done with `:Prettier`. Unlike JS, we haven't done a project-wide formatting using Prettier for HTML/CSS files yet.